### PR TITLE
fix: updating to use URI's path instead of URI's toASCIIString

### PR DESCRIPTION
**Pull request by ClaudenirFreitas**
Pull request checklist
----

- [x] Please read our [contributor guide](https://github.com/Netflix/dgs-framework/blob/master/CONTRIBUTING.md)
- [ ] Consider creating a discussion on the [discussion forum](https://github.com/Netflix/dgs-framework/discussions)
  first
- [x] Make sure the PR doesn't introduce backward compatibility issues
- [ ] Make sure to have sufficient test cases

Pull Request type
----

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
This MR is fixing [this issue](https://github.com/Netflix/dgs-framework/issues/1135).

### CODE REVIEW

**Comment by zanonena** [comment]
You have a lint problem

```
file or directory '/home/runner/work/dgs-framework/dgs-framework/graphql-dgs-spring-webflux-autoconfigure/src/test/java', not found
/home/runner/work/dgs-framework/dgs-framework/graphql-dgs-spring-webflux-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlUI.kt:62:1: Lint error > [indent] Unexpected indentation (17) (should be 16)
/home/runner/work/dgs-framework/dgs-framework/graphql-dgs-spring-webflux-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlUI.kt:63:1: Lint error > [indent] Unexpected indentation (21) (should be 20)
```

**Comment by ClaudenirFreitas** [comment]
thanks @zanonena , my bad!

**Comment by ClaudenirFreitas** [comment]
> You have a lint problem
> 
> ```
> /home/runner/work/dgs-framework/dgs-framework/graphql-dgs-spring-webflux-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlUI.kt:62:1: Lint error > [indent] Unexpected indentation (17) (should be 16)
> /home/runner/work/dgs-framework/dgs-framework/graphql-dgs-spring-webflux-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlUI.kt:63:1: Lint error > [indent] Unexpected indentation (21) (should be 20)
> ```

thanks @zanonena , my bad! 
I've just fixed it!

File: graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlConfigurer.kt [hunk 1]
```diff
@@ -47,13 +47,12 @@ class GraphiQlConfigurer(private val configProps: DgsWebfluxConfigurationPropert
     ) :
         ResourceTransformer {
         @Throws(IOException::class)
-
         override fun transform(
             exchange: ServerWebExchange,
             resource: Resource,
             transformerChain: ResourceTransformerChain
         ): Mono<Resource> {
-            if (exchange.request.uri.toASCIIString().endsWith(configProps.graphiql.path + "/index.html")) {
```
[the remark this answers is not in the document]
Comment (ClaudenirFreitas) [reply graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/GraphiQlConfigurer.kt:56]: I created [this repo to explain the issue](https://github.com/ClaudenirFreitas/dgs-issue-1135#step-by-step-to-reproduce-the-issue).

BTW, the `uri.toASCIIString` returns the query parameters as well and does not resolve the variables.

**Pull request closed without merging**